packages/database/src/migrations/20210505101731-AddSamoaCovidPercentageDashboards-modifies-data.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+import { insertObject, arrayToDbString } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_GROUP_CODES = [
+  'WS_Covid_Samoa_Country_COVID-19',
+  'WS_Covid_Samoa_District_COVID-19',
+  'WS_Covid_Samoa_Village_COVID-19',
+];
+
+const getDataBuilderConfig = doseNum => ({
+  series: {
+    value: { // The value of this key (value) IS important
+      xyz: { // The value of this key (xyz) is not important
+        operands: [
+          {
+            dataValues: [doseNum === 1 ? 'COVIDVac4' : 'COVIDVac8'],
+          },
+          {
+            dataValues: ['population_WS001'],
```

Comment:
Just wanted to note: this will sum all data for this dataElement (but it shouldn't matter as long as that is kept in mind if the population is updated)